### PR TITLE
add change close and lift NTI to use model binding

### PR DESCRIPTION
**Pull request by elielijah321**
**What is the change?**
Change the lift and close page so the data validation message appears on the page

**Why do we need the change?**
Consistencty amongst error messages

**What is the impact?**
Medium - Should be checked over by @FahadDarw 

**Azure DevOps Ticket**
[125223](https://dfe-gov-uk.visualstudio.com/Academies-and-Free-Schools-SIP/_sprints/taskboard/Concerns%20Casework/Academies-and-Free-Schools-SIP/CC/CC%20-%20Iteration%2040?workitem=125223)

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/Close.cshtml [hunk 1]
```diff
@@ -20,146 +19,55 @@
...
+                    <span class="govuk-caption-m">Case ID @Model.CaseUrn</span>
                     Close NTI
                 </h1>
 
                 <div class="govuk-warning-text">
                     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
                     <strong class="govuk-warning-text__text">
                         <span class="govuk-warning-text__assistive">Warning</span>
-                        This action cannot be reopened. Check the details are correct before closing
+                        This action cannot be reopened. Check the details are correct before lifting
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/Close.cshtml:33]: QUERY:

Should this still be closing?

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]
Just a small query on a text change

File: ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/EditableNTIPageModel.cs [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+using ConcernsCaseWork.Models;
+using ConcernsCaseWork.Models.Validatable;
+using ConcernsCaseWork.Pages.Base;
+using JetBrains.Annotations;
+using Microsoft.AspNetCore.Mvc;
+
+namespace ConcernsCaseWork.Pages.Case.Management.Action.Nti;
+
+[BindProperties]
+public class EditableNTIPageModel : AbstractPageModel
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork/Pages/Case/Management/Action/Nti/EditableNTIPageModel.cs:12]: SUGGESTION: I would call this CloseNtiBasePage

Its a page that is used by all close states of an NTI

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]

**Comment by FahadDarw** [comment]
Happy with Cypress changes.

**Pull request merged**